build/main.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+export BRANCH=$1
+export PACKAGE_JOB=$2
+export PACKAGE_UPLOAD_DIRECTORY=$3
+export PACKAGE_BUILD=$4
+
+export JOB_TO_UPLOAD=$PACKAGE_JOB
+export FLAG_FILE_URL="http://buildcache.cfengine.com/packages/$PACKAGE_JOB/$PACKAGE_UPLOAD_DIRECTORY/PACKAGES_HUB_x86_64_linux_ubuntu_16/core-commitID"
+export NO_OUTPUT_DIR=1
+
+env
+set -x
+
+# take ownersip of all files
+sudo chown -R jenkins:jenkins .
+
+export WRKDIR=`pwd`
```

Comment:
Why `export`?

Reply:
later it's used in _run_jekyll.sh script, which expects it to be set to directory where all repos are cloned (and defaults to `pwd` also). But main.sh cd's into `documentation-generator` subdir somewhere before running _run_jekyll.sh.

It's possible to remove `export` here, and `cd -` before running _run_jekyll.sh.

Or change _run_jekyll.sh to set WRKDIR to "`pwd`/..".

Which of these options sounds less terrible? :)

Reply:
`_run_jekyll.sh` should use something like `"$(dirname $0)/../"`or whatever exactly it needs.